src/lib/isPostalCode.js

```diff
@@ -11,10 +11,11 @@ const patterns = {
   AT: fourDigit,
   AU: fourDigit,
   AZ: /^AZ\d{4}$/,
+  BA: /^([7-8]\d{4}$)/,
```

Comment:
that `[7-8]` range is unnecessary, but that is a tiny style issue only.
#2141 will be dealing with a lot of these